src/expr/src/relation/mod.rs

```diff
@@ -468,18 +467,18 @@ impl MirRelationExpr {
                 typ
             }
             MirRelationExpr::Reduce {
-                input,
+                input: _,
```

Comment:
I think you could in principle delete this line (I think the `..` catches it). Happy to leave it in, ofc, but if you wanted to focus attention on the arguments that are used that might improve it.